features/support/hooks.rb

```diff
@@ -2,7 +2,12 @@
   Capybara.current_driver = :poltergeist
 end
 
-After('@javascript, @poltergeist') do
+Before('@selenium') do
+  # For JS-driven tests that Poltergeist does not handle cleanly.
+  Capybara.javascript_driver = :selenium
+end
```

Comment:
This is a hold-over from when I was testing with Selenium.  We don't need this change.